.changeset/new-paws-cross.md

```diff
@@ -0,0 +1,7 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`SuperSelect` updates including:
+- Updated `ember-power-select` from `8.2.0` to `8.6.0`
+- Added `searchFieldPosition="before-options"` to fix a11y issue in `Multiple` component
```

Comment:
How is this?

Reply:
Looks good. We can elaborate on the visual impact in Slack and the release email